### PR TITLE
Bug 1177553 - Fix failure in test_cost_control_data_alert_mobile.py

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a97a617a112e328f92fbf5362f5be7120f8ea1c4

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=58fdfd6c10c2edf20ead17285cc574b6d1219fa0

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/cost_control/test_cost_control_data_alert_mobile.py [hunk 1]
```diff
@@ -49,10 +50,9 @@ def test_cost_control_data_alert_mobile(self):
         Wait(self.marionette, timeout=60).until(lambda m: "sample_png_02.png" in m.title)
         browser.switch_to_chrome()
 
-        # get the notification bar
-        self.device.touch_home_button()
-        self.marionette.switch_to_frame()
-        self.marionette.execute_script("window.wrappedJSObject.UtilityTray.show()")
+        system = System(self.marionette)
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/cost_control/test_cost_control_data_alert_mobile.py]: :+1: No having to inject scripts is great!

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/cost_control/test_cost_control_data_alert_mobile.py [hunk 2]
```diff
@@ -64,3 +64,12 @@ def test_cost_control_data_alert_mobile(self):
         usage_view = self.marionette.find_element(*self._data_usage_view_locator)
         Wait(self.marionette, timeout=40).until(lambda m: 'reached-limit' in usage_view.get_attribute('class'),
              message='Data usage bar did not breach limit')
+        usage_view.tap()
+
+        self.wait_for_condition(lambda m: self.apps.displayed_app.name == "Usage")
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/cost_control/test_cost_control_data_alert_mobile.py]: How about using `cost_control.name`
``` python
self.wait_for_condition(lambda m: self.apps.displayed_app.name == cost_control.name)
```

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/cost_control/test_cost_control_data_alert_mobile.py [hunk 3]
```diff
@@ -64,3 +64,12 @@ def test_cost_control_data_alert_mobile(self):
...
+        self.apps.switch_to_displayed_app()
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/cost_control/test_cost_control_data_alert_mobile.py]: Why do we need to switch to the app, if we don't do anything with it?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f19534deaf326f443a41004060c173a3da5c4c59

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/cost_control/test_cost_control_data_alert_mobile.py (hunk 3 above)
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/cost_control/test_cost_control_data_alert_mobile.py]: You're right, I'll remove it.

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/cost_control/test_cost_control_data_alert_mobile.py (hunk 2 above)
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/cost_control/test_cost_control_data_alert_mobile.py]: Thanks, I'll change it.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=05b371acbba958349614577f787a40885fa20fc8

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=31b7ea3f1e85162297595c561e8b440f569bdcd1

**Pull request merged**